### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by KadeMorton**
## What does this PR do?

Updates documentation

## Why is this change important?

It updates the community information

## How to test this PR locally?

N/A

## Author's checklist

N/A

## Related issues

N/A

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -2,145 +2,108 @@
...
+Acknowledgements
 ################
 
-Are you privacy conscious user? Then yes.
+We extend our sincere appreciation to the dedicated individuals whose contributions have made Tracery a thriving open-source community and a valuable resource for the cybersecurity community!
 
-In searx we decided to double down on being privacy respecting. We are picking
-engine changes from SearxNG, but we are not implementing engine detailed
-monitoring and not adding a new UI that relies on Javascript.
+We want to extend our deepest gratitude to the original creators and maintainers of Searx, whose dedication and hard work made this powerful meta search engine possible. Without their invaluable contributions, this project would not exist.
```
Comment (jecarr) [inline README.rst:100]: For the last sentence, I was thinking

> Without their invaluable foundations, this project would not exist.

to give more weight as - at time of writing - our fork has some slight additions to Searx but it is mostly this foundation

File: README.rst [hunk 2]
```diff
@@ -2,145 +2,108 @@
...
-our homepage_.
+Creating and maintaining Tracery requires a diverse set of resources. Some of these resources are provided by Arachne Digital, the for-profit company that maintains Tracery. However, the collective is strong through diversity, which is why Arachne Digital is opening Tracery up to the community. From design and development to community building and infrastructure, here is what powers Tracery:
 
-.. _searx.space: https://searx.space
-.. _user: https://searx.github.io/searx/user
-.. _admin: https://searx.github.io/searx/admin
-.. _developer: https://searx.github.io/searx/dev
-.. _homepage: https://searx.github.io/searx
+* Design: Craft an intuitive user experience with UI/UX design and establish Tracery's unique identity through branding and visual elements.
+* Development: Skilled developers, machine learning experts, data managers, cybersecurity specialists, and quality assurance testers bring Tracery's software to life, ensuring it is robust and secure. CI/CD pipelines keep everything running smoothly.
```
Comment (jecarr) [inline README.rst:65]: Do you think we should have ML experts in this list? That applied more to Thread. Unless you see Tracery using ML in isolation one day?

File: README.rst (hunk 1 above)
Comment (KadeMorton) [reply README.rst:122]: Happy with this.

File: README.rst (hunk 2 above)
Comment (KadeMorton) [reply README.rst:87]: Please remove. I copied a lot of the content over from the Thread README.

Comment (jecarr) [reply README.rst:87]: Confirmed removing just the ML bit

**Review by jecarr — APPROVED** [review]
Thanks Kade for all the documentation work!

**Pull request merged**